components/raftstore/src/store/fsm/store.rs

```diff
@@ -534,7 +534,6 @@ impl<'a, EK: KvEngine + 'static, ER: RaftEngine + 'static, T: Transport>
             StoreTick::CompactCheck => self.on_compact_check_tick(),
             StoreTick::ConsistencyCheck => self.on_consistency_check_tick(),
             StoreTick::CleanupImportSST => self.on_cleanup_import_sst_tick(),
-            StoreTick::RaftEnginePurge => self.on_raft_engine_purge_tick(),
```

Comment:
Why remove it

Reply:
It's only for raft engine, and raft engine is not supported at v5.0.0.